src/lib.rs

```diff
@@ -42,17 +45,20 @@ pub fn depythonize<T>(_py: Python, _obj: PyObject) -> T {
 pub struct PythonizerError(PyErr);
 
 impl ser::Error for PythonizerError {
-    fn custom<T>(_msg: T) -> Self
+    fn custom<T>(msg: T) -> Self
     where
         T: std::fmt::Display,
     {
-        todo!()
+        let s: String = format!("{}", msg);
+        PythonizerError {
+            0: PyErr::new::<PyException, _>(s)
+        }
```

Comment:
`std::fmt::Display` gives us a blanket impl of `std::string::ToString`. Also I prefer the `PyException::new_err` constructors personally:

```suggestion
        PythonizerError(PyException::new_err(s.to_string()))
```